docs/index.md

```diff
@@ -1,4 +1,15 @@
-# Select you language
+# CQtDeployer
 
-* [Рус](ru/Home)
-* [Eng](en/Home)
\ No newline at end of file
+Here you can find documentation for the CQtDeployer in different languages. It describes:
+
+* Information on how to use this deployment Tool. (How to create installers with cqtdeployer)
+
+* Main description (description of the functionality and capabilities of the program)
+
+* Description of recent changes (changelog)
```

Comment:
```suggestion
* Information on how to use this deployment Tool. (How to create installers with cqtdeployer)
* Main description (description of the functionality and capabilities of the program)
* Description of recent changes (changelog)
```

Hello!

It would be better if you put these items in one list.

You can view and compare the rendered list with the one below "Сhoose your language to read the CQtDeployer documentation" in https://github.com/QuasarApp/CQtDeployer/blob/9310505e9925d0075f127240d24c6cf09201261e/docs/index.md .

Reply:
Okey. Thanks.